wrapper/virt_v2v_wrapper.py

```diff
@@ -63,9 +63,9 @@ def prepare_command(data, v2v_caps, agent_sock=None):
             '--no-copy',
         ])
     else:
-        v2v_args.append(data['vm_name'])
```

Comment:
For SSH we've always kept the `ssh://` URI in `vm_name`, why this change?

Reply:
Because I introduce a `vmware_uri` field in https://github.com/ManageIQ/manageiq/pull/20011. It will be more consistent with what we have for VDDK.

Reply:
I've updated the documentation accordingly.